src/Api/Resources/ItemWishListResource.php

```diff
@@ -45,8 +46,11 @@ public function __construct(Request $request, ApiResponse $response, ItemWishLis
      */
     public function index():Response
     {
-        $itemWishList = $this->itemWishListService->getItemWishList();
-        return $this->response->create($itemWishList, ResponseCode::OK);
+        /** @var ItemListService $itemListService */
+        $itemListService = pluginApp(ItemListService::class);
+        $items = $itemListService->getItemList(ItemListService::TYPE_WISH_LIST, null, null, null, null);
```

Comment:
```suggestion
        $items = $itemListService->getItemList(ItemListService::TYPE_WISH_LIST);
```

Die Parameter `$id`, `$sorting`, `$maxItems` und `$crossSellingRelation` haben alle einen default-wert und müssen daher nicht explizit mit `null` angegeben werden.